modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-sagemaker.adoc

```diff
@@ -58,7 +58,7 @@ image::{dir}/sagemaker-config-2.PNG[Create lifecycle configuration,align="center
 
 ** On create script: 
 +
-[source, bash, id="sagemaker-first-config", role="content-editable emits-gtm-events"]
```

Comment:
Thanks for this one. But I think I will open a new PR for fixing this.

Reply:
Got it!